inspec.yml

```diff
@@ -1,12 +1,117 @@
-# encoding: utf-8
-
+---
 name: windows-baseline
 title: DevSec Windows Security Baseline
+summary: An InSpec Compliance Profile for CIS Microsoft Windows Server 2012R2 and 2016 RTM (Release 1607) Benchmark Level 1 and 2 - Member Server and Domain Controller. It contains additional controls from the MS technet.
+version: 2.0.0
```

Comment:
This is great, since we ensure we do not break existing customers. We should release the latest master before we merge this PR.

Reply:
definitely. i agree